js/ui/components/clubhouse.js

```diff
@@ -111,6 +111,10 @@ var ClubhouseWindowTracker = new Lang.Class({
 
         this._windowActor = actor;
 
+        // We need to activate the window directly as otherwise it won't be focused because
+        // of having a DOCK hint type
+        actor.meta_window.activate(global.get_current_time());
```

Comment:
Calling `activate()` on the side component window sounds like a good idea, but not every time this callback is received... I am not sure under which circumstances the `restacked` signal is received, and always activating a window in response to what is a signal emitted potentially when another window is activated does not feel like a robust solution to me. (Even though I understand that ill side effects are protected by the `this._windowActor == actor` check above at the moment.)

Since I offered an easy way to solve the dragging issue from the app side in the ticket, I'd like to explore that avenue further before committing this workaround.